INSTALL.md

```diff
@@ -1,13 +1,13 @@
 # dattobd INSTALL
 
-## From Repositories
+## Installing From Repositories
 We recommend that you install the kernel module from Datto's repositories. Datto provides repos for x86-64 editions of RHEL/CentOS, Fedora, openSUSE, SUSE Linux Enterprise, Debian, and Ubuntu LTS.
 
 ### RHEL/CentOS
 The repository install package `datto-el-rpm-release` is available for EL6+.
 ```bash
 sudo yum localinstall https://cpkg.datto.com/datto-rpm/repoconfig/datto-el-rpm-release-$(rpm -E %rhel)-latest.noarch.rpm
-sudo yum install dkms-dattobd dattobd-utils
+sudo yum install kernel-headers-$(uname -r) dkms-dattobd dattobd-utils
```

Comment:
Strictly speaking, we should probably request `kernel-devel-$(uname -r)` here, but this should be installed as dependency of DKMS. I'm not sure why this is needed?